src/csf/transformCsf.ts

```diff
@@ -60,15 +61,26 @@ const makePlayTest = (
   ];
 };
 
-const makeDescribe = (key: string, tests: t.Statement[]): t.Statement | null => {
+const makeDescribe = (
+  key: string,
+  tests: t.Statement[],
+  beforeEachBlock?: t.ExpressionStatement
+): t.Statement | null => {
+  const blockStatements = beforeEachBlock ? [beforeEachBlock, ...tests] : tests;
   return t.expressionStatement(
     t.callExpression(t.identifier('describe'), [
       t.stringLiteral(key),
-      t.arrowFunctionExpression([], t.blockStatement(tests)),
+      t.arrowFunctionExpression([], t.blockStatement(blockStatements)),
     ])
   );
 };
 
+const makeBeforeEach = (beforeEachPrefixer?: FilePrefixer) => {
```

Comment:
should be required b/c you're testing for falseness outside of the function call

Reply:
great catch!